modules/runner/src/integrationTest/java/org/apache/ignite/internal/benchmark/SqlBenchmark.java

```diff
@@ -52,11 +53,14 @@
 @BenchmarkMode(Mode.AverageTime)
 @OutputTimeUnit(TimeUnit.MILLISECONDS)
 @SuppressWarnings({"WeakerAccess", "unused"})
-public class SqlOneNodeBenchmark extends AbstractOneNodeBenchmark {
+public class SqlBenchmark extends AbstractMultiNodeBenchmark {
```

Comment:
The javadoc is inaccurate about the `on single node cluster`.

Reply:
done